sdcm/nemesis.py

```diff
@@ -1716,9 +1713,42 @@ def disrupt_network_start_stop_interface(self):  # pylint: disable=invalid-name
         finally:
             self.target_node.remoter.run("sudo /sbin/ifup eth1")
 
+    def disrupt_grow_shrink_cluster(self):
+        timeout_before_shrink = self.tester.params.get("nemesis_interval")
```

Comment:
better to use self.interval (and init it)

Reply:
done